test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/LanguageServer/PeekDefinitionTests.cs

```diff
@@ -35,10 +29,10 @@ public class PeekDefinitionTests
         /// Test get definition for a table object with active connection
         /// </summary>
         [Fact]
-        public async Task GetValidTableDefinitionTest()
+        public void GetValidTableDefinitionTest()
```

Comment:
yes default is onPrem (2016) and master

---
In reply to: [97459380](https://github.com/Microsoft/sqltoolsservice/pull/218#discussion_r97459380) [](ancestors = 97459380)